native/screens/Applicants/Applicants.jsx

```diff
@@ -87,21 +81,56 @@ const RoleList = ({ applicants, navigation, role, currProject }) => {
             }}
           >
             <View>
-              <View>
+              <View style={styles.applicantInfo}>
                 <Text>IMAGE</Text>
```

Comment:
can we replace this text with a sample image for now? Can just be a round view container with a dark background like in the figma